cpp/src/plasma/common.cc

```diff
@@ -27,8 +28,88 @@ namespace fb = plasma::flatbuf;
 
 namespace plasma {
 
+namespace {
+
+const char kErrorDetailTypeId[] = "plasma status detail";
```

Comment:
Turn this into `plasma::PlasmaStatusDetail`?

Reply:
Yep